### PR TITLE
Add ship info to science

### PR DESCRIPTION
**Pull request by foesi**
Make a ship description only visible when scanned. Also added another description property to ships which only appears if the ship is deep scanned. This is currently only possible for ships. Maybe this should be extended to ships and stations. For other objects it is maybe worthwhile that the description can be seen all the time.

### CODE REVIEW

**Comment by czenker** [comment]
The feature of this pull requests was implemented in a recent version: https://github.com/daid/EmptyEpsilon/blob/master/CHANGELOG.md#2017-11-03

So this might be closed I guess.

File: src/menus/autoConnectScreen.cpp [hunk 1]
```diff
@@ -124,7 +124,7 @@ bool AutoConnectScreen::isValidShip(int index)
 
     for(auto it : ship_filters)
     {
-        if (it.first == "solo")
+        if (it.first.compare("solo"))
```
Comment (daid) [inline src/menus/autoConnectScreen.cpp]: Why did you change this? And even more importantly, it's broken now.

**Comment by foesi** [comment]
Sorry. I did a mistake. In Java there is a difference between equals and == and I did not look it up for c++.
I reverted it so now at least this problem should no longer exist.

**Comment by foesi** [comment]
@czenker said this was already implemented.
